tests/mtest/src/main/scala/tests/BaseSuite.scala

```diff
@@ -58,16 +51,18 @@ class BaseSuite extends munit.FunSuite with Assertions {
   def getExpected(
       default: String,
       compat: Map[String, String],
-      scalaVersion: String = this.scalaVersion
+      scalaVersion: String
   ): String = {
     val postProcess = compatProcess
-      .get(scalaBinary(scalaVersion))
-      .orElse(compatProcess.get(scalaVersion))
+      .collect {
+        case (ver, process) if scalaVersion.startsWith(ver) => process
```

Comment:
Changed this to startsWith, so that we can use "0." -> for Scala 3. We can also just add a condition for major version, opinions?

Reply:
startsWith looks okay, (collect + headOption == collectFirst)

Reply:
Thanks, fixed!